scripts/jira/cloner.py

```diff
@@ -20,36 +37,41 @@ def is_original_issue(issue):
     return True
 
 def get_clone_by_issues(issue, connection):
-    l = []
+    """Returns a list of clone issues for the given issue."""
+    lst = []
     for link in issue.fields.issuelinks:
         if link.type.name != 'Cloners':
             continue
         if hasattr(link, 'inwardIssue'):
-            l.append(connection.issue(link.inwardIssue.key))
-    return l
+            lst.append(connection.issue(link.inwardIssue.key))
```

Comment:
I wonder if we actually want that naming rule applied. Maybe we should have a conversation about it on slack?

Reply:
Reverted the changes and turned off the rule.